contents/blog/how-to-measure-product-engagement.md

```diff
@@ -16,104 +17,94 @@ tags:
   - Guides
 ---
 
-Product engagement is the most important factor when it comes to driving retention and revenue for your business. Though it seems easy to view success through a financial lens, revenue alone is rarely a clear indicator of a sustainable product. 
+Everyone wants high product engagement. It's one of those things that sounds obviously good, like "growth" or "traction." But what does it actually mean, and how do you measure it? 
 
-In this article we’ll take a look at why strong customer engagement metrics are vital to the success of your product, how to define engagement for your platform, and how to use analytics tools to measure and build on the results.
+In this blog, we'll explain what it is, common ways people measure it, and what metrics startups should actually focus on.
 
 **Contents:**
 
 - [What is product engagement?](#what-is-product-engagement)
 - [What are product engagement metrics?](#what-are-product-engagement-metrics)
-  - [Common engagement metrics](#common-engagement-metrics)
-- [What is the best way to measure product engagement?](#what-is-the-best-way-to-measure-product-engagement)
-  - [Active users](#active-users)
-  - [Stickiness](#stickiness)
-  - [Adoption](#adoption)
-- [What tools are good for tracking product engagement?](#what-tools-are-good-for-tracking-product-engagement)
+- [Common engagement metrics](#common-product-engagement-metrics)
+- [How to measure product engagement for startups](#how-to-measure-product-engagement-for-startups)
+- [Quick recap](#quick-recap)
 - [Further reading](#further-reading)
 
-> This article is part of our [PostHog Academy series](/tracks) where we explain the fundamentals of product analytics.
-
 ## What is product engagement?
```

Comment:
We want to get to the "meat" of the blog as quick as possible. A common strategy we use to do this (especially when we have a "what is X" section) is drop the intro paragraphs and just make the "what is X" section the intro. 

I think you can combine this section with the intro.